x-pack/plugins/maps/public/actions/store_actions.js

```diff
@@ -413,6 +414,7 @@ export function endDataLoad(layerId, dataId, requestToken, data, meta) {
       meta,
       requestToken
     });
+    dispatch(setLayerDataLoadErrorStatus(layerId, null));
```

Comment:
Would it make more sense to clear the layer error on `startDataLoad ` instead of `endDataLoad `?

Reply:
we would get a jittering of the error-message if it persists for multiple calls.

Reply:
Can you put that in a comment in the `endDataLoad ` method so we don't move it later and forget why its here